### PR TITLE
Fw 4259 migrate confusables

### PR DESCRIPTION
**Pull request by caforbes**
### Description of Changes
* Added an importer + tests for confusables. The importer converts the CSV and all its rows to a list which are stored in 1 JSONField on a site Alphabet.
* Adjusted test skipping for import resources - continue to test models that we will import in supplementary migrations for 1.0 launch
* Added searching by site name to the Alphabet admin (JSON makes the page really long)

### Checklist
- README / documentation has been updated if needed
- [X] PR title / commit messages contain Jira ticket number if applicable
- [X] Tests have been updated / created if applicable
- [X] Admin Panel has been updated if models have been added or modified
- Migrations have been updated and committed if applicable
- Team Postman workspace has been updated if applicable

### Reviewers Should Do The Following:
- [ ] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

**Review by sarahfirstvoices — APPROVED** [review]
Love the admin and tests updates!

File: firstvoices/backend/resources/characters.py [hunk 1]
```diff
@@ -38,3 +43,27 @@ class Meta:
...
+            for in_form, out_form in zip(dataset["in_form"], dataset["out_form"]):
+                mapper = {"in": in_form, "out": out_form}
+                list_of_mappers.append(mapper)
+
+            alphabet.input_to_canonical_map = list_of_mappers
+            alphabet.save()
+        return super().before_import(dataset, using_transactions, dry_run, **kwargs)
+
+    def skip_row(self, instance, original, row, import_validation_errors=None):
+        "Never import individual rows."
```
Comment (bronson-c) [inline firstvoices/backend/resources/characters.py:68]: [minor] Is this meant to be a comment? It seems like just a string without an assignment.

**Review by bronson-c — APPROVED** [review]
LGTM 👍 , just one minor question.

File: firstvoices/backend/tests/test_resources/test_character_resource.py [hunk 2]
```diff
@@ -201,3 +211,63 @@ def test_import_base_data(self):
...
+
+class TestAlphabetConfusablesImport:
+    @staticmethod
+    def build_table(data: list[str]):
+        headers = [
+            # these headers should match what is produced by fv-nuxeo-export tool
+            "out_form,in_form,site",
+        ]
+        table = tablib.import_set("\n".join(headers + data), format="csv")
+        return table
```
[the remark this answers is not in the document]
Comment (caforbes) [reply firstvoices/backend/tests/test_resources/test_character_resource.py:224]: I think everything I would think to test is taken care of by the exporter. I'll leave it for now but we should add error and edge cases if we want to reuse this importer beyond the nuxeo migration context.
